### PR TITLE
feat(php): release 10.21.0.11

### PR DESCRIPTION
**Pull request by ZNeumann**

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-21-0-11.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+subject: PHP agent
+releaseDate: '2024-05-20'
+version: 10.21.0.11
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.21.0.11'
+features: []
+bugs: ['end guzzle's external segment for rejected request']
```
Comment (ZNeumann) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-21-0-11.mdx:7]:
```suggestion
bugs: ['end guzzle external segments for rejected request']
```

**Comment by ZNeumann** [comment]
We are planning on releasing before noon PDT today

**Comment by ZNeumann** [comment]
We have released and this PR can be merged as soon as convenient

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-21-0-11.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+version: 10.21.0.11
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.21.0.11'
+features: []
+bugs: ['end guzzle external segments for rejected request']
+security: ['Upgrade golang to 1.22']
+---
+## New Relic PHP agent v10.21.0.11
+
+### Bug fixes
+* fix(agent): end guzzle's external segment for rejected request (https://github.com/newrelic/newrelic-php-agent/pull/883)
```
Comment (ally-sassman) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-21-0-11.mdx:13]:
```suggestion
* fix(agent): End guzzle's external segment for rejected request (https://github.com/newrelic/newrelic-php-agent/pull/883)
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**